src/logger.js

```diff
@@ -0,0 +1,26 @@
+import colors from 'colors/safe';
+
+var colorList = ['blue', 'yellow', 'magenta', 'cyan'];
+
+var enabled = !!process.env.DEBUG;
+
+export default function logger (command, ...messages) {
+  if (enabled) {
+    var color = colorList[command.length % colorList.length];
```

Comment:
@Paxa Yes, but what is the point of changing the color for different functions.  In fact, what is the point in having colors at all.  Personally I find colors annoying in CLI tools and having seemingly random colors assigned to random logs would confuse more people than it helps.

For instance "yellow" would be interpreted as a warning., blue as info. Etc.  The easiest thing to do is to leave colors out completely